cmd/ocm-backplane/managedJob/createManagedJob.go

```diff
@@ -175,11 +175,13 @@ func createJob(client BackplaneApi.ClientInterface) (*BackplaneApi.Job, error) {
 	// create job request
 	createJob := BackplaneApi.CreateJobJSONRequestBody{
 		CanonicalName: &options.canonicalName,
-		Parameters: &jobParams,
+		Parameters:    &jobParams,
 	}
 
-	// call create end point
-	resp, err := client.CreateJob(context.TODO(), options.clusterID, createJob)
+	// call create endpoint and handle Backplane-Warning header
+	resp, err := utils.HandleBackplaneWarningHeader(func() (*http.Response, error) {
+		return client.CreateJob(context.TODO(), options.clusterID, createJob)
+	})
```

Comment:
Do we need a util function to handle it ? What if just get a warning msg and log it as a warning

```
resp, err := client.CreateJob(context.TODO(), options.clusterID, createJob)
if warningMsg := resp.Header.Get("Backplane-Warning"); warningMsg != ""  {
   logger.Warnf("Backplane-Warning: %s", warningMsg)
}
```

Reply:
@samanthajayasinghe - Sorry, I am bit confused. As mentioned in your first review comment, I got 2 points: 

- What if use the logger to print warnings **based on debug level**
- It is better to handle it via **commonplace** and apply to all subcommands.

My latest commit just trying to match the suggestions above: 

- HandleBackplaneWarningHeader function checks for the Backplane-Warning header in the response, and logs it **at different levels based on the current log level** (You also provided `jobs.go` which defined the flag for log level)
- I am trying to make the function **HandleBackplaneWarningHeader** reusable for any HTTP request. As long as other parts of cli use this function to handle HTTP responses, the warning logging behavior will be consistent across all backplane API calls.

I think we should have a quick sync directly to make sure I am not over thinking anything.